params/governance_params.go

```diff
@@ -53,6 +53,11 @@ const (
 	Policy
 	CommitteeSize
 	UnitPrice
+	LowerBoundBaseFee
+	UpperBoundBaseFee
+	GasTarget
+	MaxBlockGasUsedForBaseFee
+	BaseFeeDenominator
```

Comment:
We should add them at the end of the itoa list. 
Those values are `iota` type and used for governance voting. It means that the itoa values of previous votings are contained in previous block headers. So, this change can cause chain forks during the synchronizing of old blocks.

Reply:
@aidan-kwon Is it really? As far as I know the string names are written into headers, not integer keys. Database stores string names too. Anyway I agree that enums should be appended at the end as a general caution.

Reply:
Ahh, It could have been very critical.
I fixed it, thanks.

Reply:
Oh @blukat29 your review appeared lately. I think we need to discuss more this.

Reply:
@blukat29 Uhm.. You're right. Votes in the header use plain string instead of a number. If Klaytn doesn't store governance data with this itoa values, it's gonna be okay. It is safe to add in the end, though.

Reply:
I got it, Thanks any way.